exchanges/kucoin/kucoin_test.go

```diff
@@ -3025,37 +3025,40 @@ func TestSubscribeBatches(t *testing.T) {
 func TestSubscribeBatchLimit(t *testing.T) {
```

Comment:
Commentary above talks about 300 still

Reply:
Oop. Thanks!

Fixed https://github.com/gbjk/gocryptotrader/commit/d9162c7cd6

I was looking for references to 300 in the non-test files.